.goreleaser.yaml

```diff
@@ -6,8 +6,6 @@ before:
 
 builds:
   - binary: oasis
-    env:
-      - CGO_ENABLED=0
```

Comment:
I believe the only reason for having this was that this config was copied from the [oasis-web3-gateway](https://github.com/oasisprotocol/oasis-web3-gateway) that had this set.

Reply:
as long as folks aren't using this in a no-libc container or something :D